internal/utils/eszip/build.ts

```diff
@@ -0,0 +1,44 @@
+import * as path from "https://deno.land/std@0.127.0/path/mod.ts";
```

Comment:
Is there any documentation on when and how to update these scripts?

Reply:
I simplified this to only have `eszip/build.ts` as an embed in f01cb72.  That should make updating modules easier.